labelbox/schema/model.py

```diff
@@ -18,23 +18,26 @@ class Model(DbObject):
     name = Field.String("name")
     model_runs = Relationship.ToMany("ModelRun", False)
 
-    def create_model_run(self, name) -> "ModelRun":
+    def create_model_run(self, name, config) -> "ModelRun":
```

Comment:
Adding a mandatory config here will be a breaking change for existing customer scripts that call this function. Better to set a default value of None here, and handle that accordingly
```suggestion
    def create_model_run(self, name, config=None) -> "ModelRun":
```